packages/plugin-kit/src/createSliceMachinePluginRunner.ts

```diff
@@ -142,7 +142,8 @@ export class SliceMachinePluginRunner {
 				)(resolve);
 				plugin = raw.default || raw;
 			} catch (error) {
-				if (import.meta.env.DEV) {
+				// Only log in development and when plugin is unavailable through native plugins library (to prevent noise)
+				if (import.meta.env.DEV && !(resolve in this._nativePlugins)) {
```

Comment:
❓ Why kind of noise we are talking here. I tend to like having errors always display, so you don't miss a problem.

Reply:
This kind of noise: https://github.com/prismicio/slice-machine/actions/runs/6575577367/job/17863112184?pr=1178, the unit test logs are unreadable because all tests are relying on native plugins, but maybe there's a better way?

To my understanding, we added this line to make clearer while developing what prevented a plugin from loading which is unrelated to plugin testing with native plugins(?)

Reply:
I agree with your last question. But are we sure it will be the only error that can be thrown here?
If yes, your change is perfect. 
If not, maybe we could add a condition to not display it specifically in test (process.env.VITEST for example)?

Reply:
I'm ok for either of the two personally 👍

Reply:
Done 👌